docs/source/en/model_doc/gpt_neox.mdx

```diff
@@ -0,0 +1,71 @@
+<!--Copyright 2021 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# GPT-NeoX
+
+## Overview
+
+The GPT-NeoX-20B model was released in the [EleutherAI/gpt-neox](https://github.com/EleutherAI/gpt-neox) repository.
+It is a GPT-2 like causal language model trained on the
+[Pile](https://pile.eleuther.ai/) dataset.
+
+Development of the model was led by Sid Black, Stella Biderman and Eric Hallahan, and the model was trained with generous the support of [CoreWeave](https://www.coreweave.com/).
+More details about the model can be found in the [manuscript](https://arxiv.org/abs/2204.06745).
+
```

Comment:
We also like to include tips when using the model. Working on it, do you have any tips to give to the users that would like to use it? Since it's a very big model, mentioning parallelization would be interesting I think.

Mentioning the rotary embeddings could be interesting if you think it applies here, and finally the tokenizer seems to have been modified in order to handle spaces smarter than the GPT-2 tokenizer would; this likely results in quite a few improvements that I would mention here as well.

Reply:
@LysandreJik I would agree about parallelization, but we do plan on submitting more (smaller) models with this class as well. Do you think that it would be appropriate to mention things that are model-specific here?

Reply:
Indeed, good question. I would mention it for the big model, but maybe it is more appropriate in the model card that can be hosted [here](https://huggingface.co/EleutherAI/gpt-neox-20b) then. Looking forward to the smaller models! :)